Sources/LanguageServerProtocol/SupportTypes/TestItem.swift

```diff
@@ -68,7 +68,7 @@ public struct TestItem: ResponseType, Equatable {
     tags: [TestTag]
   ) {
     self.id = id
-    self.label = label
+    self.label = id == label ? self.id : label
```

Comment:
Why do we need this change? I don’t think it’s doing anything, right?

Reply:
Good catch, forgot to remove this.